Automattic/wp-calypso
# Add Stats Yearly (annual) product to 'calypso-products' package.

This PR adds the Jetpack Stats yearly (annual) product to the `calypso-products` package, making the annual product available to all of Calypso




## Proposed Changes

* Added necessary constants to the `calypso-products` package.
* The annual Stats product was added to the wpcom backend in: code-D118168

## Testing Instructions



- If code-D118168 is not merged yet, you will first need to apply patch code-D118168 to your sandbox and sandbox `public-api.wordpress.com`, and `define( 'USE_STORE_SANDBOX' );` in `0-sandbox.php` file.
- Checkout this PR and run `yarn start`.
- Go to: http://calypso.localhost:3000/checkout/jetpack/jetpack_stats_yearly
- Verify the yearly and monthly billing term dropdown is showing on the checkout page.
- Verify the "Included features" in the right side column are showing.
- See the before and after screenshots below, checkout should look like the after screenshot.

**BEFORE** | **AFTER**
--- | ---
![Markup 2023-08-08 at 08 16 23](https://github.com/Automattic/wp-calypso/assets/11078128/1862a716-a733-4ea2-b4f6-ad34f9ba4a1e) | ![Markup 2023-08-08 at 08 12 54](https://github.com/Automattic/wp-calypso/assets/11078128/104e61df-51f0-43c4-ba08-d80d4512aee8)

.




## Pre-merge Checklist
